### ISSUE TITLE
triggers as axis

### ISSUE THREAD

**Issue by polymorphicengine** [danomatika/joyosc#11]
there is the following code in `GameController.cpp`:

```
if(axis == "lefttrigger" || axis == "righttrigger") {
				return buttonPressed(axis, (event->caxis.value > 0 ? 1 : 0));
			}
```

with a comment that for some devices the triggers are recognised as axis.. I don't think this is a bug and  i find it very useful to have the full range of sensitivity on the triggers. Removing the above lines results in a working solution.

Maybe you could add an option where one can choose how triggers should be interpreted?

**Comment by danomatika** [comment]
This is now a config file option, per controller in joyosc 0.5.0: https://github.com/danomatika/joyosc/blob/master/data/example_config.xml#L53

**Comment by danomatika** [comment]
>I don't think this is a bug and i find it very useful to have the full range of sensitivity on the triggers.

I think it depends on the device. On macOS, a the trigger buttons on a Playstation 3 controller only return 0 or 32767 -> essentially a binary button. This is the behavior I actually needed, in this case, so hence the current default.

Other controllers might give variable responses, but I haven't personally tested one.

**Comment by danomatika** [comment]
This was implemented in joyosc 0.5.0
